diff --git forkSrcPrefix/app/build.gradle forkDstPrefix/app/build.gradle
index e0ec580f86404f2eda00bca72ad3f19c1b744973..8e5bcea2a21820887b94f9fbd5b9b9a60b5608c3 100644
--- forkSrcPrefix/app/build.gradle
+++ forkDstPrefix/app/build.gradle
@@ -5,7 +5,7 @@ plugins {
 
 // 多渠道打包配置
 apply from: rootProject.file('flavors.gradle')
-apply from: rootProject.file('signings.gradle')
+// apply from: rootProject.file('signings.gradle')
 apply from: rootProject.file('buildRelease.gradle')
 
 // 1. 配置 App Bundle (.aab) 的基础文件名
@@ -46,6 +46,23 @@ android {
         versionName siyuanVersionName
     }
 
+    signingConfigs {
+        siyuanConfig {
+            storeFile file("../keystore/debug.jks")
+            storePassword "android"
+            keyAlias "androiddebugkey"
+            keyPassword "android"
+        }
+
+        debug {
+            storeFile = file("../keystore/debug.jks")
+            storePassword System.getenv("KEYSTORE_PASSWORD")
+            keyAlias "debug"
+            keyPassword System.getenv("KEYSTORE_PASSWORD")
+        }
+    }
+
+
     buildFeatures {
         buildConfig true
     }
@@ -65,11 +82,13 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
+            signingConfig signingConfigs.debug
+
             applicationIdSuffix ".debug"
 
             minifyEnabled false
 
-            resValue "string", "app_name", "SiYuan-Debug"
+            resValue "string", "app_name", "SiYuan"
             resValue "string", "app_package_name", "org.b3log.siyuan.debug"
 
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
