diff --git forkSrcPrefix/app/build.gradle forkDstPrefix/app/build.gradle
index 3a62b3e68cab5e05e3e5a3c393e751a3f9e78f5f..cb051027937c496ef920545b62aef4ed39cb8bef 100644
--- forkSrcPrefix/app/build.gradle
+++ forkDstPrefix/app/build.gradle
@@ -30,6 +30,15 @@ android.applicationVariants.configureEach { variant ->
 }
 
 android {
+    signingConfigs {
+        debug {
+            storeFile = file("../keystore/debug.jks")
+            storePassword System.getenv("KEYSTORE_PASSWORD")
+            keyAlias "debug"
+            keyPassword System.getenv("KEYSTORE_PASSWORD")
+        }
+    }
+
     namespace 'org.b3log.siyuan'
     compileSdk 34
 
@@ -56,9 +65,10 @@ android {
         debug {
             applicationIdSuffix ".debug"
             minifyEnabled false
-            resValue "string", "app_name", "SiYuan-Debug"
+            resValue "string", "app_name", "SiYuan"
             resValue "string", "app_package_name", "org.b3log.siyuan.debug"
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
         }
     }
 
