diff --git forkSrcPrefix/kernel/api/setting.go forkDstPrefix/kernel/api/setting.go
index e0c248e8d586cf1e64de32b935e38b513b0b896e..b4da536b24163c790b559787f49cc25aff20ec22 100644
--- forkSrcPrefix/kernel/api/setting.go
+++ forkDstPrefix/kernel/api/setting.go
@@ -432,18 +432,53 @@ func getCloudUser(c *gin.Context) {
 	ret := gulu.Ret.NewResult()
 	defer c.JSON(http.StatusOK, ret)
 
-	arg, ok := util.JsonArg(c, ret)
-	if !ok {
-		return
-	}
+	// arg, ok := util.JsonArg(c, ret)
+	// if !ok {
+	// 	return
+	// }
+
+	// t := arg["token"]
+	// var token string
+	// if nil != t {
+	// 	token = t.(string)
+	// }
+	// if err := model.RefreshUser(token); nil != err {
+	// 	ret.Code = 1
+	// 	ret.Msg = err.Error()
+	// 	return
+	// }
+	// ret.Data = model.Conf.User
+
+	user := &conf.User{
+			UserId: "0",
+			UserName: "_",
+			UserAvatarURL: "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
+			UserHomeBImgURL: "",
+			UserTitles: []*conf.UserTitle{},
+			UserIntro: "",
+			UserNickname: "",
+			UserCreateTime: "29991231 00:00:00",
+			UserSiYuanProExpireTime: -1,
+			UserToken: "token",
+			UserTokenExpireTime: "32503593600",
+			UserSiYuanRepoSize: 0,
+			UserSiYuanPointExchangeRepoSize: 0,
+			UserSiYuanAssetSize: 0,
+			UserTrafficUpload: 0,
+			UserTrafficDownload: 0,
+			UserTrafficAPIGet: 0,
+			UserTrafficAPIPut: 0,
+			UserTrafficTime: 0,
+			UserSiYuanSubscriptionPlan: 0,
+			UserSiYuanSubscriptionStatus: 0,
+			UserSiYuanSubscriptionType: 1,
+			UserSiYuanOneTimePayStatus: 0,
+	}
+
+	model.Conf.User = user
+	model.Conf.Save()
 
-	t := arg["token"]
-	var token string
-	if nil != t {
-		token = t.(string)
-	}
-	model.RefreshUser(token)
-	ret.Data = model.Conf.User
+	ret.Data = user
 }
 
 func logoutCloudUser(c *gin.Context) {
